src/de/mod.rs

```diff
@@ -653,6 +653,17 @@ mod tests {
         assert!(from_str::<[i32; 2]>("[0, 1,]").is_err());
     }
 
+    #[test]
+    fn tuple_variant() {
+        #[derive(Debug, Deserialize, PartialEq)]
+        enum TupleVariant {
+            Add(i32, i32),
+            Sub(i32, i32),
+        }
+        assert_eq!(from_str(r#"{"Add":[3,4]}"#), Ok(TupleVariant::Add(3, 4)));
+        assert_eq!(from_str(r#"{"Sub":[5,6]}"#), Ok(TupleVariant::Sub(5, 6)));
+    }
```

Comment:
Could you add a test that verifies this is serialized the same way as [serde_json](https://crates.io/crates/serde_json) does (for both `Nothing` and `Empty`)?

Reply:
Done.